daphne/src/lib.rs

```diff
@@ -236,6 +242,24 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// DAP version used for a task.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub enum DapVersion {
+    Unknown,
+    Draft01,
+    Draft02,
+}
```

Comment:
* I think the JSON serialization of this enum should match the endpoints (https://serde.rs/variant-attrs.html#rename)
* No need to add variant for  02 since we don't support this version yet.
* Serde has a feature that allows one of the variants to be the "default" if none of the other tags are matched.
* Prefer snake case to camel case.
```suggestion
pub enum DapVersion {
    #[serde(rename = "v01")]
    Draft01,
    #[serde(other)]
    Unknown,
}
```

Reply:
I didn't know about this feature in serde. This makes the code cleaner.